holidayextras/ui-toolkit
# Adding CircleCI

#### What does this PR do? (please provide any background)
We may be moving from TravisCI to CircleCI - This PR add in the circle.yml file and has the same configuration as already exists in Travis.

#### What tests does this PR have?
No tests, however i pushed a working branch, a broken branch and a fixed branch to check everything worked as expected.

#### How can this be tested?
- Pull down the UI-Toolkit
- Create a branch
- Push the branch and check everything is okay in CircleCI
- Break a couple of tests an push the branch
- Check CircleCI fails (should show as failed in build chat in slack)
- Fixed the tests again and push
- Check CircleCI passes (should also see fix in build chat in slack)

#### Screenshots / Screencast
<img width="621" alt="screen shot 2015-11-06 at 14 23 42" src="https://cloud.githubusercontent.com/assets/778942/10999088/01aaf074-8492-11e5-9bb8-e58e778bb4c5.png">

#### What gif best describes how you feel about this work?
![circle](https://media.giphy.com/media/GVjU3YQLBhMGI/giphy.gif)

---
#### Developer Definition of Done/Quality Checklist (for PR author to complete BEFORE code review):

#### Software Engineer or Developer review:

#### Software Engineer or project guru review:
